app/assets/stylesheets/modules/search.scss

```diff
@@ -117,6 +117,10 @@ header .navbar-form .input-group .input-group-btn {
   height: $search-box-form-height;
 }
 
+.search-q.form-control.tt-hint {
+  color: white;
```

Comment:
Can we try setting it to zero opacity, similar to the way it's done in https://github.com/projectblacklight/blacklight/pull/2807/files

Reply:
Hello! Please see my initial comment above. When I set 0 opacity, it looks like this: 
<img width="1145" alt="image" src="https://user-images.githubusercontent.com/80173363/195185857-85060b80-7566-44cb-96bf-09f8ec0ed653.png">

I tried a number of different selectors but wasn't able to isolate just the autocomplete text--any suggestions?

Reply:
Ah! sorry I  missed that.